### ISSUE TITLE
I follow the tutorial to deploy a node.js web app on Azure step by step, but it failed when az webapp up. 

### ISSUE THREAD

**Issue by Lohengrinnn** [Azure/azure-cli#11080]
https://docs.microsoft.com/en-us/learn/modules/host-a-web-app-with-azure-app-service/6-exercise-deploy-your-code-to-app-service?pivots=node

I follow the tutorial to deploy a node.js web app on Azure step by step, but it failed when az webapp up. 

## Describe the bug

**Command Name**
`az webapp up`

**Errors:**
```
'int' object has no attribute 'upper'
Traceback (most recent call last):
python3.6/site-packages/knack/cli.py, ln 206, in invoke
    cmd_result = self.invocation.execute(args)
cli/core/commands/__init__.py, ln 603, in execute
    raise ex
cli/core/commands/__init__.py, ln 661, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
cli/core/commands/__init__.py, ln 652, in _run_job
    cmd_copy.exception_handler(ex)
...
cli/command_modules/appservice/custom.py, ln 2993, in webapp_up
    create_app_service_plan(cmd, rg_name, plan, _is_linux, False, sku, 1 if _is_linux else None, location)
cli/command_modules/appservice/custom.py, ln 1390, in create_app_service_plan
    sku = _normalize_sku(sku)
cli/command_modules/appservice/utils.py, ln 20, in _normalize_sku
    sku = sku.upper()
AttributeError: 'int' object has no attribute 'upper'
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az webapp up --name {} --resource-group {} --plan {} --sku {} --location {}`

## Environment Summary
```
Linux-4.15.0-1061-azure-x86_64-with-debian-stretch-sid
Python 3.6.5
Shell: bash

azure-cli 2.0.76

```

**Comment by auspic7** [comment]
I am having this problem, too.

**Comment by auspic7** [comment]
Duplicate of #11080

**Comment by auspic7** [comment]
Duplicate of #11076

**Comment by it-support5** [comment]
I followed basic example from https://docs.microsoft.com/de-de/azure/app-service/app-service-web-get-started-html and get the same error message.

**Comment by cf-sewe** [comment]
I followed basic example from https://docs.microsoft.com/de-de/azure/app-service/app-service-web-get-started-html and get the same error message.

**Comment by mcampos19cr** [comment]
I also reported this using a similar tutorial

**Comment by Juliehzl** [comment]
dupe of #11100

**Comment by panchagnula** [comment]
Fixed by https://github.com/Azure/azure-cli/pull/11019

**Comment by Lohengrinnn** [comment]
> Fixed by #11019

it still doesn't work.
